Cannot find module 'react-native-paper/babel' when generating production build for android

**Without Expo** & Using - "react-native": "0.60.4", "react-native-paper": "^3.0.0-alpha.3",

_babel.config.js--_
```javascript
module.exports = {
  presets: ['module:metro-react-native-babel-preset'],
  env: {
    production: {
      plugins: ['react-native-paper/babel'],
    },
  },
};
```

when trying to generate APK on windows system getting this error. tried on other projects using RN paper "^3.0.0-alpha.3", got same error

![rn paper issus](https://user-images.githubusercontent.com/26733312/63611154-e40b0700-c5f8-11e9-9ae6-6000f82c53f9.png)

@jaysbytes unfortunately it did not worked.

```
denis@DESKTOP-GS9VKQF MINGW64 /f/repos/inqool/spu/app/node_modules/react-native-paper (master)
$ yarn prepare
yarn run v1.17.3
$ bob build && node ./scripts/generate-mappings.js
ℹ Building target module
ℹ Cleaning up previous build at lib\module
ℹ Compiling 102 files in src with babel
✓ Wrote files to lib\module
ℹ Building target typescript
ℹ Cleaning up previous build at lib\typescript
ℹ Generating type definitions with tsc
✖ Errors found when building definition files.
../../tsconfig.json(6,3): error TS5053: Option 'declaration' cannot be specified with option 'isolatedModules'.
../../tsconfig.json(10,3): error TS5053: Option 'emitDeclarationOnly' cannot be specified with option 'noEmit'.

bob.js build

build files for publishing

Options:
  --help     Show help                                                 [boolean]
  --version  Show version number                                       [boolean]

Error: Failed to build definition files.
    at build (F:\repos\inqool\spu\app\node_modules\react-native-paper\node_modules\@react-native-community\bob\lib\targets\typescript.js:63:11)        
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

@jaysbytes 
I got this on `3.0.0-alpha.4`

```
error: bundling failed: Error: Cannot find module '../../dist/mappings.json'
Require stack:
- /Users/ripzery/workspace/projects/work/wallet/node_modules/react-native-paper/lib/module/babel/index.js
- /Users/ripzery/workspace/projects/work/wallet/node_modules/react-native-paper/babel.js
- /Users/ripzery/workspace/projects/work/wallet/node_modules/@babel/core/lib/config/files/plugins.js
- /Users/ripzery/workspace/projects/work/wallet/node_modules/@babel/core/lib/config/files/index.js
- /Users/ripzery/workspace/projects/work/wallet/node_modules/@babel/core/lib/index.js
- /Users/ripzery/workspace/projects/work/wallet/node_modules/metro/src/JSTransformer/worker.js
- /Users/ripzery/workspace/projects/work/wallet/node_modules/metro/src/DeltaBundler/Worker.js
- /Users/ripzery/workspace/projects/work/wallet/node_modules/jest-worker/build/workers/processChild.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:623:15)
    at Function.Module._load (internal/modules/cjs/loader.js:527:27)
    at Module.require (internal/modules/cjs/loader.js:681:19)
    at require (internal/modules/cjs/helpers.js:16:16)
    at Object.<anonymous> (/Users/ripzery/workspace/projects/work/wallet/node_modules/react-native-paper/lib/module/babel/index.js:1:14)
    at Module._compile (internal/modules/cjs/loader.js:774:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:785:10)
    at Module.load (internal/modules/cjs/loader.js:641:32)
    at Function.Module._load (internal/modules/cjs/loader.js:556:12)
 BUNDLE  [android, prod, minified] ./index.js ░░░░░░░░░░░░░░░░ 0.0% (0/1)::ffff:127.0.0.1 - - [09/Sep/2019:09:16:32 +0000] "GET /index.bundle?platform=android&dev=false&minify=true HTTP/1.1" 500 - "-" "okhttp/3.12.1"
 BUNDLE  [android, prod, minified] ./index.js ░░░░░░░░░░░░░░░░ 0.0% (0/1), failed.

 BUNDLE  [android, prod, minified] ./index.js ░░░░░░░░░░░░░░░░ 0.0% (0/1)::ffff:127.0.0.1 - - [09/Sep/2019:09:16:36 +0000] "GET /index.bundle?platform=android&dev=false&minify=true HTTP/1.1" 500 - "-" "okhttp/3.12.1"
error: bundling failed: Error: Cannot find module '../../dist/mappings.json'
Require stack:
- /Users/ripzery/workspace/projects/work/wallet/node_modules/react-native-paper/lib/module/babel/index.js
- /Users/ripzery/workspace/projects/work/wallet/node_modules/react-native-paper/babel.js
- /Users/ripzery/workspace/projects/work/wallet/node_modules/@babel/core/lib/config/files/plugins.js
- /Users/ripzery/workspace/projects/work/wallet/node_modules/@babel/core/lib/config/files/index.js
- /Users/ripzery/workspace/projects/work/wallet/node_modules/@babel/core/lib/index.js
- /Users/ripzery/workspace/projects/work/wallet/node_modules/metro/src/JSTransformer/worker.js
- /Users/ripzery/workspace/projects/work/wallet/node_modules/metro/src/DeltaBundler/Worker.js
- /Users/ripzery/workspace/projects/work/wallet/node_modules/jest-worker/build/workers/processChild.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:623:15)
    at Function.Module._load (internal/modules/cjs/loader.js:527:27)
    at Module.require (internal/modules/cjs/loader.js:681:19)
    at require (internal/modules/cjs/helpers.js:16:16)
    at Object.<anonymous> (/Users/ripzery/workspace/projects/work/wallet/node_modules/react-native-paper/lib/module/babel/index.js:1:14)
    at Module._compile (internal/modules/cjs/loader.js:774:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:785:10)
    at Module.load (internal/modules/cjs/loader.js:641:32)
    at Function.Module._load (internal/modules/cjs/loader.js:556:12)
```

--- a/package.json
+++ b/package.json
@@ -6,7 +6,6 @@
   "types": "lib/typescript/src/index.d.ts",
   "files": [
     "lib",
-    "dist",
     "babel.js"
   ],
   "sideEffects": false,

--- a/scripts/generate-mappings.js
+++ b/scripts/generate-mappings.js
@@ -6,7 +6,7 @@ const types = require('babel-types');
 const parser = require('@babel/parser');
 
 const root = path.resolve(__dirname, '..');
-const output = path.join(root, 'dist/mappings.json');
+const output = path.join(root, 'lib/mappings.json');
 const source = fs.readFileSync(path.resolve(root, 'src', 'index.tsx'), 'utf8');
 const ast = parser.parse(source, {
   sourceType: 'module',
@@ -20,7 +20,7 @@ const ast = parser.parse(source, {
 });
 
 const relative = (value /* : string */) =>
-  path.relative(root, path.resolve(path.dirname(root, 'lib', 'module'), value));
+  path.relative(root, path.resolve(path.dirname(require.resolve('..')), value));
 
 const mappings = ast.program.body.reduce((acc, declaration, index, self) => {
   if (types.isExportNamedDeclaration(declaration)) {

--- a/src/babel/__fixtures__/rewrite-imports/output.js
+++ b/src/babel/__fixtures__/rewrite-imports/output.js
@@ -1,11 +1,11 @@
 /* eslint-disable prettier/prettier */
 import { Text } from 'react-native';
-import PaperProvider from "react-native-paper/../core/Provider";
-import BottomNavigation from "react-native-paper/../components/BottomNavigation";
-import Button from "react-native-paper/../components/Button";
-import FAB from "react-native-paper/../components/FAB/FAB";
-import Appbar from "react-native-paper/../components/Appbar/Appbar";
-import * as Colors from "react-native-paper/../styles/colors";
+import PaperProvider from "react-native-paper/lib/module/core/Provider";
+import BottomNavigation from "react-native-paper/lib/module/components/BottomNavigation";
+import Button from "react-native-paper/lib/module/components/Button";
+import FAB from "react-native-paper/lib/module/components/FAB/FAB";
+import Appbar from "react-native-paper/lib/module/components/Appbar/Appbar";
+import * as Colors from "react-native-paper/lib/module/styles/colors";
 import { NonExistent, NonExistentSecond as Stuff } from 'react-native-paper';
-import { ThemeProvider } from "react-native-paper/../core/theming";
-import { withTheme } from "react-native-paper/../core/theming";
+import { ThemeProvider } from "react-native-paper/lib/module/core/theming";
+import { withTheme } from "react-native-paper/lib/module/core/theming";

--- a/src/babel/index.js
+++ b/src/babel/index.js
@@ -1,10 +1,13 @@
-const mappings = require('../../dist/mappings.json');
+const mappingsPath =
+  process.env.NODE_ENV === 'test'
+    ? `../../lib/mappings.json`
+    : `../../mappings.json`;
+const mappings = require(mappingsPath);
 
 const SKIP = Symbol('SKIP');
 
 module.exports = function rewire(babel) {
   const t = babel.types;
-
   return {
     visitor: {
       ImportDeclaration(path) {

